CONTRIBUTING.md

```diff
@@ -61,21 +74,35 @@ Now you can access the cluster using kubectl, e.g.: `kubectl get ns`.
 To set up a containerized Splunk instance, you can use either podman or docker.
 Follow these steps:
 
-1. Build and Run the Splunk instance by running the below cmd:
+1. Copy the splunk template file and replace the template's variables
```

Comment:
Its annoying to have to do this, its better if we pre-create a file for manual use and commit it to the repo. For manual use we can also assume fixed port numbers rather then random ones.

If we want to get fancy we can put together a quick cli tool for this and then add a comment somewhere so that it is invoked via `go generate`. The result should still be committed to the repo.

Reply:
Seems like a good nice-to-have, I'll stick with the yaml for now but we can open a ticket for it

Reply:
fixed on #43